lab_3_keywords_textrank/main.py

```diff
@@ -120,7 +134,9 @@ def _learn_indices(self, tokens: tuple[str, ...]) -> None:
             tokens : tuple[str, ...]
                 sequence of string tokens
         """
-        pass
+        for token in tokens:
+            self._word2id[token] = 1000 if len(self._word2id.values()) == 0 else max(self._word2id.values()) + 1
```

Comment:
you count `1000 if len(self._word2id.values()) == 0 else max(self._word2id.values()) + 1` every iteration, but can only once